cmd/generator/main.go

```diff
@@ -87,6 +91,20 @@ var generatorCmd = &cobra.Command{
 			"mattermost-plugin-nps",
 		}
 
+		type iconURL struct {
+			URL    string
+			Inline bool
```

Comment:
Why do you want to support both passing the URL to the server and passing data to the server? Why not stick with one way?

Reply:
As discussed, sticking with the single way.